### DISCUSSION TITLE
conference proceedings

### DISCUSSION THREAD

**Discussion by Joaoblsepulveda** [withanage/conference#12] [General]
Can i use this plugin to export a XML file as a conference proceedings instead of a journal ?

**Comment by diegoabadan** [comment]
We will publish a plugin for this, see:
https://forum.pkp.sfu.ca/t/ojs-export-xml-as-a-conference-proceedings/83368/4

**Comment by Joaoblsepulveda** [comment]
Appreciate it!

I'm trying to submit a conference proceeding paper to crossref but had to
do it manually which could take some time if we had a lot of submissions to
deposit.

Thank you, I will keep my eyes on the plugin, let me know if there will be
any funding or something like that.

Em qui., 23 de nov. de 2023 às 10:52, Diego Abadan ***@***.***>
escreveu:

**Reply by diegoabadan** [reply]
The plugin is available:
https://forum.pkp.sfu.ca/t/ojs-export-xml-as-a-conference-proceedings/83368/6

**Comment by Joaoblsepulveda** [comment]
Thanks, I'll check it out!

Em qui., 30 de nov. de 2023 às 17:41, Diego Abadan ***@***.***>
escreveu:
